cockatrice/src/dlg_settings.cpp

```diff
@@ -269,8 +282,9 @@ void GeneralSettingsPage::retranslateUi()
     defaultUrlLabel.setText(tr("Primary download URL:"));
     fallbackUrlLabel.setText(tr("Fallback download URL:"));
     urlLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(WIKI_CUSTOM_PIC_URL).arg(tr("How to set a custom picture url")));
-    clearDownloadedPicsButton.setText(tr("Reset/Clear Downloaded Pictures"));
-    updateNotificationCheckBox.setText(tr("Notify when new client features are available"));
+    clearDownloadedPicsButton.setText(tr("Reset/clear downloaded pictures"));
+    updateReleaseChannelLabel.setText(tr("Update channel"));
+    updateNotificationCheckBox.setText(tr("Notify when a new version is available"));
```

Comment:
Do we have any logic for automatic update checks yet?! First time I hear about that. :)